lib/js/components/DatePickers/DateRangePicker/DateRangePicker.stories.ts

```diff
@@ -0,0 +1,146 @@
+import DateRangePicker from './DateRangePicker.vue';
```

Comment:
i think that we decide that new stories should be written in new api template like in https://github.com/bethinkpl/design-system/blob/7c73389e632d2788018bb262f2af5e443ffcb7aa/lib/js/components/Well/Well.stories.ts#L2-L1

I can't find any official confirmation, so I'll leave it as **OPTIONAL** and I'll add this on retro